app/src/main/kotlin/com/wire/android/ui/home/conversationslist/bottomsheet/HomeSheetContent.kt

```diff
@@ -114,7 +124,7 @@ internal fun HomeSheetContent(
             },
             {
                 if (conversationSheetContent.conversationTypeDetail is ConversationTypeDetail.Private) {
-                    if (conversationSheetContent.conversationTypeDetail.isBlockable) {
+                    if (conversationSheetContent.conversationTypeDetail.blockingState == BlockingState.NOT_BLOCKED) {
```

Comment:
Shouldn't we have "unblock" button here if the user is blocked? 🤔

Reply:
yes, but i decided it's a part of another task "Unblock user" ;P